metals/src/main/scala/scala/meta/internal/metals/testProvider/TestSuitesProvider.scala

```diff
@@ -419,6 +419,11 @@ final class TestSuitesProvider(
       currentTarget.testSymbols
         .readOnlySnapshot()
         .toList
+        // sort the symbols lexically so that symbols with the same fullyQualifiedName
+        // will be grouped, and the class will come before companion object (i.e.
+        // `a.b.WordSpec#` < `a.b.WordSpec.`). This ensures that the class is put into the cache
+        // instead of the companion object.
+        .sortBy(_._1)
```

Comment:
Can you change it to `sortBy { case (symbol, _) => symbol }`?

Reply:
@kpodsiad Thanks for the review! I'll update the PR based on the comment.